src/main/resources/static/javascript/mainApp/grouping.controller.js

```diff
@@ -1582,7 +1582,12 @@
                 scope: $scope,
                 backdrop: "static",
                 keyboard: false,
-                ariaLabelledBy: "sync-dest-modal"
+                ariaLabelledBy: "sync-dest-modal",
+                controller: "SyncDestModalController",
+                resolve: {
+                    isSynced: isSyncDestOn,
+                    syncDestDescription: () => $scope.selectedSyncDest.description,
```

Comment:
Does syncDestDescription need to be a function? I'm not sure.

Reply:
Angular complains about unknown provider if it isn't a function. I have no idea why isSynced works fine even though it isn't a function.

Reply:
Maybe Angular doesn't like that $scope.selectedSyncDest.description is a $scope variable? Or maybe you need to wrap this resolve object in parenthesis? 
So possibly,
```
resolve: ({
    isSynced: isSyncDestOn,
    syncDestDescription: $scope.selectedSyncDest.description
})
```

Reply:
Still not sure why. I tried changing it to be a JS const and also wrapping it in parenthesis, but it still complaied about unknown provider

Reply:
That's ok, we can leave it as is.